src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +26,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
```

Comment:
The `getAllProductsFromShoppingCart` method returns a hardcoded list of products. In a production environment, this data should be fetched from a database based on the user's shopping cart.